pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -906,6 +906,13 @@ class Build {
                 }
             }
 
+            try {
+                context.echo("Cleaning any old build workspace from ${env.WORKSPACE}/workspace, as git clean -fdx can take a very long time to do the same.")
+                context.sh(script: "rm -rf ${env.WORKSPACE}/workspace")
```

Comment:
Can we have a comment here with an issue link on why this is necessary for brevity (like the windows ones above)? Also, you should also be able to use cleanws instead of the script but that's just a suggestion, I won't be sad if you don't include that

Reply:
cleanWs() cleans the whole existing repo, as per @sxa comment we should be leaving the existing cached git repo's

Reply:
i've added a comment pointing at the issue